index.js

```diff
@@ -8,10 +9,24 @@ const parser = require("./src/parser");
 const printDocToDebug = require("./src/doc-debug").printDocToDebug;
 
 function parse(text, opts) {
-  if (opts.parser === "flow") {
-    return parser.parseWithFlow(text, opts.filename);
+  const parseFunction = opts.parser === "flow"
+    ? parser.parseWithFlow
+    : parser.parseWithBabylon;
+
+  try {
+    return parseFunction(text);
+  } catch (error) {
+    const loc = error.loc;
+
+    if (loc) {
+      error.codeFrame = codeFrame(text, loc.line, loc.column + 1, {
+        highlightCode: true
```

Comment:
Thanks. I failed to notice that you mentioned rollup-plugin-node-<strong>globals</strong> (instead of rollup-plugin-node-<strong>builtins</strong>) the second time -.-

I’ve got it all working now!